lib/actions/ui.js

```diff
@@ -166,3 +171,32 @@ export function windowMove () {
     });
   };
 }
+
+export function openFile (path) {
+  return (dispatch, getState) => {
+    dispatch({
+      type: UI_OPEN_FILE,
+      effect () {
+        stat(path, (err, stats) => {
+          if (!err) {
```

Comment:
I handled this error only to be sure that an error is never thrown, but it's really unlikely that you could open a file that doesn't exists, and if you find a way to do so I can't immagine what you should expect :P 
However, how could I send a notification to the user? Should I import utils/notify.js or there is an error event i could emit?